packages/quantic/force-app/examples/main/lwc/utils/utils.js

```diff
@@ -0,0 +1,24 @@
+// @ts-ignore
+import getHeadlessConfiguration from '@salesforce/apex/HeadlessController.getHeadlessConfiguration';
```

Comment:
Is it really the best file to have this in?
At least perhaps a caseAssistUtils.js instead of a generic utils.js?